core/src/lib.rs

```diff
@@ -291,12 +339,29 @@ impl Schema {
     pub fn identifies_attribute(&self, x: &NamespacedKeyword) -> bool {
         self.get_entid(x).map(|e| self.is_attribute(e)).unwrap_or(false)
     }
+
+    /// Returns an symbolic representation of the schema suitable for applying across Mentat stores.
+    pub fn to_edn_value(&self) -> edn::Value {
+        edn::Value::Vector((&self.schema_map).into_iter()
```

Comment:
Can you make sure you can call `to_edn_value` twice in a row on the same `Schema`? E.g., a test that doing so twice yields the same result.